core/UrlHelper.php

```diff
@@ -39,7 +39,7 @@ class UrlHelper
     protected static function inArrayMatchesRegex($test, $patterns): bool
     {
         foreach ($patterns as $val) {
-            if (@preg_match($val, null) === false) {
+            if (@preg_match($val, '') === false) {
```

Comment:
Can we add that change to [`VisitsExcluded`](https://github.com/matomo-org/matomo/blob/80b20d8dd69079524738f403ddc37daf2fca26fb/core/Tracker/VisitExcluded.php#L374) as well?